src/Faucet.sol

```diff
@@ -43,11 +73,8 @@ contract Faucet is Ownable, IFaucet {
      * @dev Claim via meta tx on behalf of a new account by the owner
      * @param _signatureData Signature data
      */
-    function claimOnBehalf(IFaucet.SignatureData calldata _signatureData)
-        external
-        onlyOwner
-        onlySignerVerified(_signatureData)
-    {
+    function claimOnBehalf(IFaucet.SignatureData calldata _signatureData) external onlySignerVerified(_signatureData) {
```

Comment:
```suggestion
    function claimKintoETH(IFaucet.SignatureData calldata _signatureData) external onlySignerVerified(_signatureData) {
```

Reply:
you can keep the same name of the function but just change the argument and the caller can assume that if you are passing a signature it's because you are claiming on behalf of someone

Reply:
changing, good idea